srcalc/src/main/java/gov/va/med/srcalc/web/view/VariableSummary.java

```diff
@@ -78,18 +100,23 @@ public void visitMultiSelect(final MultiSelectVariable variable)
         {
             fTypeName = (variable.getDisplayType() == DisplayType.Radio) ?
                     "Radio Button" : "Drop-Down";
+            fOptionString = String.format("[%s]", Joiner.on(", ").join(variable.getOptions()));
         }
         
         @Override
         public void visitDiscreteNumerical(final DiscreteNumericalVariable variable)
         {
             fTypeName = "Discrete Numerical";
+            fOptionString = String.format("Range: %s Options: [%s]", 
```

Comment:
I thought it would be useful because you see the entire range of the variable. Even though the user has to use an option as the value, you at least know what range you are dealing with and that could be better information for the user since "> 1.5" might not fully qualify how large the category is.